scipy/stats/stats.py

```diff
@@ -3999,13 +3999,16 @@ def fisher_exact(table, alternative='two-sided'):
     --------
     chi2_contingency : Chi-square test of independence of variables in a
         contingency table.
+    contingency.odds_ratio : Compute the odds ratio for a 2x2 contingency
+        table.
 
     Notes
     -----
-    The calculated odds ratio is different from the one R uses. This scipy
-    implementation returns the (more common) "unconditional Maximum
-    Likelihood Estimate", while R uses the "conditional Maximum Likelihood
-    Estimate".
+    The calculated odds ratio is different from the one R uses. This
+    implementation returns the "unconditional maximum likelihood estimate",
+    while R uses the "conditional maximum likelihood estimate".  To compute
```

Comment:
nit: different number of spaces between ". This" and ".  To compute"

Reply:
Done.

Reply:
Fortunately Sphinx renders these as single spaces : )